camera_k: [399.04169696001776, 0.0, 368.06125222560473, 0.0, 398.9109964300084, 213.71148849469188, 0.0, 0.0, 1.0]
camera_d: [-0.010545391389310708, -0.006435344397266451, 0.06313970558713196, -0.036126706892504905]
voc_file: "/workspaces/voxgraph_melodic_ws/src/coxgraph/coxgraph/config/briskOKVIS_k10_L6_voc_v1.yml.gz"

  # 0: MONOCULAR, 1: STEREO, 2: RGBD
sensor: 0

publish_landmarks: true

map_frame: "world"

min_dist_m: 0.0
min_yaw_rad: 0.0
min_local_score: 1.0
min_kf_n: 0

tracking:
  min_features_num: 2000
